engine/apps/alerts/incident_log_builder/incident_log_builder.py

```diff
@@ -120,25 +163,22 @@ def _get_resolution_notes(self) -> "RelatedManager['ResolutionNote']":
             "created_at"
         )
 
-    def get_incident_escalation_plan(self, for_slack=False):
-        """
-        Generates dict with escalation plan with timedelta as keys and list with plan lines as values
-        :param for_slack: (bool) add user slack id to plan line or not
-        :return:
-        """
-        incident_escalation_plan = dict()
-        incident_escalation_plan = self._add_invitation_plan(incident_escalation_plan, for_slack=for_slack)
+    def get_escalation_plan(self, for_slack: bool = False) -> FlattendEscalationPlan:
+        escalation_plan: EscalationPlan = dict()
+        escalation_plan = self._add_invitation_plan(escalation_plan, for_slack=for_slack)
+
         if not self.alert_group.acknowledged and not self.alert_group.is_silenced_forever:
-            incident_escalation_plan = self._add_escalation_plan(incident_escalation_plan, for_slack=for_slack)
-        final_incident_escalation_plan = self._finalize_escalation_plan_dict(incident_escalation_plan)
-        return final_incident_escalation_plan
```

Comment:
saw an opportunity to (continue) removing references to "incident" (I found this is confusing wrt an "incident" from Grafana Incident).. updated references to this function